### PR TITLE
Replacing iconv with iconv-lite

### PR DESCRIPTION
**Pull request by florian-senn**
Other changes may be caused by JetBrains auto-formatting

### CODE REVIEW

File: lib/mvgApi.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 const cheerio = require("cheerio");
 const request = require("request");
 const Line = require("./Line");
-const iconv = require("iconv");
+const iconv = require("iconv-lite");
```
Comment (Lynbarry) [inline lib/mvgApi.js]: Add a const for the default value here
`const defaultApi = "https://www.mvg-live.de/ims/dfiStaticAuswahl.svc";`

File: lib/mvgApi.js [hunk 2]
```diff
@@ -15,36 +15,36 @@ const iconv = require("iconv");
  */
 function getDepartures(station, options) {
```
Comment (Lynbarry) [inline lib/mvgApi.js]: Change the signature to 
`function getDepartures(station, options, apiUrl = defaultApi)`

File: lib/mvgApi.js [hunk 3]
```diff
@@ -15,36 +15,36 @@ const iconv = require("iconv");
...
 
-    let transitParams = buildTransitParams(options);;
+    let transitParams = buildTransitParams(options);
 
-    const requestUrl = 'http://www.mvg-live.de/ims/dfiStaticAuswahl.svc?haltestelle='.concat(station).concat(transitParams);
+    const requestUrl = 'https://cors-anywhere.herokuapp.com/https://www.mvg-live.de/ims/dfiStaticAuswahl.svc?haltestelle='.concat(station).concat(transitParams);
```
Comment (Lynbarry) [inline lib/mvgApi.js]: Change this to:
`const requestUrl = apiUrl.concat('?haltestelle').concat(station).concat(transitParams);`

**Review by Lynbarry — CHANGES_REQUESTED** [review]
Maybe we shouldn't have the url hardcoded inside the getDepartures-Function at all.

I would propose adding a third parameter to the getDepartures Method, "apiUrl" with a default value of the direct mvg url.

This way it is possible to adjust the endpoint to your needs, but it also defaults to the original url. I'll comment my proposed changes where they would occur.

**Comment by Lynbarry** [comment]
Would changing the signature work for you?

I can also do this myself, but I wanted to try out the github pullrequest process to be honest...;)

File: lib/mvgApi.js [hunk 4]
```diff
@@ -15,36 +15,36 @@ const iconv = require("iconv");
  */
```
Comment (Lynbarry) [inline lib/mvgApi.js:17]: Documentation for the apiUrl Parameter.

**Comment by florian-senn** [comment]
Hey Lynbarry, your requested changes are done. Forgot about the URL, after I encountered CORS-errors. The proposed changes are a good feature!

**Review by Lynbarry — APPROVED** [review]

**Pull request merged**